src/wp-includes/blocks/navigation.php

```diff
@@ -846,7 +852,7 @@ function block_core_navigation_typographic_presets_backcompatibility( $parsed_bl
  */
 function block_core_navigation_parse_blocks_from_menu_items( $menu_items, $menu_items_by_parent_id ) {
```

Comment:
This deprecated function is still being used in `render_block_core_navigation`. Can that be updated as well before this is merged?

Reply:
Good spot! I'm afraid I'll leave it for Beta 2 since we're already one hour behind schedule for Beta 1, and it doesn't seem like a total blocker 😅